versioned_docs/version-21.10/releases/centreon-core.md

```diff
@@ -211,6 +211,19 @@ Release date: `February 23, 2022`
 - Fixed an issue causing BAM Business Activities (best status) to remain in an OK state when the OK KPIs were removed
 - Refactored the BAM Business activities downtimes inheritance mechanism so that they are properly inherited and not duplicated anymore.
 
+> You may have remaining downtimes because they were duplicated from a KPI downtime that ended before the update. In t
```

Comment:
```suggestion
> You may have remaining downtimes because they were duplicated from a KPI downtime that ended before the update. In that case, you will have to apply this procedure.
```